### DISCUSSION TITLE
I hope someone can help me

### DISCUSSION THREAD

**Discussion by BeshoyAdelSaad** [community/community#27695] [Other Feature Feedback, Questions, & Ideas]
I have many problems, I am new in the field of programming and I feel that there are many things I do not know in this field, I entered this field because it attracted me.
I spent 10 months watching videos of HTML, CSS, JavaScript, PHP, MYSQL, and finally Python.
Now I loved Python, but I didn't know what to specialize in and there are many things I don't understand in Python. I also feel that I don't have friends in this field to tell them about any problem I face.
I hope someone can help me

**Comment by airtower-luna** [comment]
If you need help with a problem it's best to describe the problem directly, so people who might be able to help can see it. No-one can say whether they can answer your question without knowing the question. :wink: 

You can post here (the "General" category should be right for that, see #21279), and if you'd like a place focused on Python there's also the [Python Forum](https://discuss.python.org/). :snake:

**Comment by cdefgah** [comment]
Start small, don't try to learn everything at once. As you have chosen Python, start with plain basics, syntax, flow control, etc. Important thing is that you should not only read/watch something, you should **do** things, you have just read/watched. It involves different neural structures and helps to memorize things better. To get this even better, get a blog on a free platform, for example on blogspot.com (if you have a gmail account, you already have blogspot account too, because it is Google platform). And post things you have learned in your blog, like you are explaining something to someone. It will significantly improve the learning process.

A link for the start: https://pynative.com/python-basic-exercise-for-beginners/

Good luck.

**Reply by BeshoyAdelSaad** [reply]
Thanks for your reply and I will try to do it, but the real problems are two things:
First I'm not very good at English and sometimes enter programming websites and face issues in understanding many words, but I try to translate every word that I don't know it.
Second: I don't have time and should be working in this field as soon as possible because I need money.

**Reply by cdefgah** [reply]
To translate texts to your native language you can use Google Translator: https://translate.google.com/

This translator performs better, but has fewer language options: https://www.deepl.com/translator

What I would do in your case:

1. To resolve financial issues, I would find any work that is applicable according to the current skills
2. Collect information about open junior positions, and study things that are required
3. As I get more confident in my programming skills I would apply to a junior position in the programming field.

Additional links:

How to write a resume for a junior developer:
https://www.freecodecamp.org/news/how-to-write-an-awesome-junior-developer-resume-in-a-few-simple-steps-316010db80ec/

Python junior developer resume samples:
https://zety.com/blog/python-developer-resume-example

Free python courses for beginners: 
https://www.freecodecamp.org/news/learn-python-free-python-courses-for-beginners/
